barretenberg/cpp/src/barretenberg/bb/main.cpp

```diff
@@ -145,23 +146,18 @@ bool proveAndVerifyGoblin(const std::string& bytecodePath,
                           const std::string& witnessPath,
                           [[maybe_unused]] bool recursive)
 {
-    info("Construct constraint_system and witness.");
```

Comment:
You could probably keep these and use vinfo instead, so that they are only ran when the user uses -v